File: Java/src/main/java/com/sk/model/WorkItemsDTO.java
package com.sk.model;

public class WorkItemsDTO {
    private int ID;
    private String workItemType;
    private String title;
    private String assignedTo;
    private String state;
    private String tags;

    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public String getWorkItemType() {
        return workItemType;
    }

    public void setWorkItemType(String workItemType) {
        this.workItemType = workItemType;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getAssignedTo() {
        return assignedTo;
    }

    public void setAssignedTo(String assignedTo) {
        this.assignedTo = assignedTo;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getTags() {
        return tags;
    }

    public void setTags(String tags) {
        this.tags = tags;
    }
} 

File: Dotnet/Chat/src/WorkItems/WorkItemsDTO.cs
namespace WorkItems;

public class WorkItemsDTO
{
    public int ID { get; set; }
    public required string WorkItemType { get; set; } = string.Empty;
    public required string Title { get; set; } = string.Empty;
    public required string AssignedTo { get; set; } = string.Empty;
    public required string State { get; set; } = string.Empty;
    public string Tags { get; set; } = string.Empty;
}
